integr8ly/integreatly-operator
# MGDAPI-6044 Bump Keycloak Client to v0.1.14

# Issue link
Jira: https://issues.redhat.com/browse/MGDAPI-6044

# What
Keycloak Client is bumped to new release v0.1.14

# Verification steps

- provision a cluster

- `checkout this branch`

- run `make cluster/prepare/local`

- run `make code/run`

- verify that the Integreatly operator is running without errors
